vignettes/approx-inference.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Approximate Bayesian inference in `epidist`"
+description: "..."
+output: 
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+    number_sections: true
+pkgdown:
+  as_is: true
+# csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Getting started with epidist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+---
+
+```{r setup, include=FALSE}
+# exclude compile warnings from cmdstanr
+knitr::opts_chunk$set(
+  fig.path = "figures/epidist-",
+  cache = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  error = FALSE
+)
+```
+
+# Background
+
+The `epidist` package uses Bayesian inference to estimate delay distributions and other quantities.
+Doing Bayesian inference amounts to approximating the posterior distribution of each parameter in the statistical model^[See a work-in-progress vignette about the model structure.].
+
+By default, `epidist` uses the No-U-Turn Sampler [NUTS; @hoffman2014no] Hamiltonian Monte Carlo (HMC) algorithm to approximate the posterior distribution.
+NUTS is an example of a broader class of Markov chain Monte Carlo (MCMC) methods which work by simulating from a Markov chain with the targetted posterior distribution as stationary distribution.
+When MCMC algorithms are run for sufficiently many iterations, and have "reached convergence", the samples can be treated as being being drawn from the posterior distribution and used to compute relevant posterior quantities such as expectations.
+A drawback of MCMC methods like NUTS, is that they can be quite computational intensive, especially for complex models or large data.
+
+The `epidist` package is built using `brms` [@brms], which stands for "Bayesian Regression Models using Stan", where Stan [@carpenter2017stan] is a probabilistic programming language.
+NUTS is not the only inference algorithm implemented in Stan.
+By relying on `brms` (and Stan) these additional inference algorithms are also available in `epidist`.
+As such, if you are using `epidist` and having difficulties (for example you may have a long runtime, or your chains are failing to converge) using the default NUTS algorithm, you may want to consider an alternative.
```

Comment:
I think if your changes aren't converging then none of the other methods are likely to work (unless its due to run time).

Reply:
(I assume chains here rather than changes.)

Yes agree in part. Could push back here on what does it mean "to be due to runtime".